iothub/service/src/DigitalTwin/Models/InvokeDigitalTwinCommandOptions.cs

```diff
@@ -16,26 +16,31 @@ public class InvokeDigitalTwinCommandOptions
         public string Payload { get; set; }
 
         /// <summary>
-        /// The timeout before the command request will fail if the request takes too long to reach the device.
-        /// This timeout may happen if the target device is not connected to the cloud or if the cloud fails to deliver
-        /// the request to the target device in time. If this value is set to 0 seconds, then the target device must be online
-        /// when this command request is made.
+        /// The amount of time given to the service to connect to the device.
```

Comment:
You can copy this almost verbatim to the DirectMethodRequest.cs since it has these same two configurable timeouts

Reply:
done